apps/condo/domains/ticket/schema/TicketChange.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Generated by `createschema ticket.TicketChange 'ticket:Relationship:Ticket:CASCADE;'`
+ */
+
+const { Text, Relationship, Integer, Select, Checkbox, DateTimeUtc, CalendarDay, Decimal, Password, File } = require('@keystonejs/fields')
+const { Json } = require('@core/keystone/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/TicketChange')
+
+
+const TicketChange = new GQLListSchema('TicketChange', {
+    schemaDoc: 'Incremental changes of Ticket',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        ticket: {
+            schemaDoc: 'Related ticket, whose change is logged in this entity',
+            type: Relationship,
+            ref: 'Ticket',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+    },
+    plugins: [uuided(), tracked()],
```

Comment:
versioned?

Reply:
Why we need `versioned`?
TicketChange is an immutable entity.

Reply:
Added in `3d0ebcf8` for consistency with other schemas, as you asked.